docs/example.md

```diff
@@ -37,7 +37,7 @@ Each connector is typically based on a "class" such as ODBC or JDBC, and provide
 
 ![]({{ site.baseurl }}/assets/pce-connect-pane.png)
 
-## ![2]({{ site.baseurl }}/assets/pce-2.png) 
+## ![2]({{ site.baseurl }}/assets/pce-2.png) Tableau Custom Dialog File
 
 The Tableau Custom Dialog file (.tcd) is optional. By default, your connecor inherits a connection dialog from its parent (defined by <span style="font-family: courier new">superclass</span> You can use the TCD file to customize the connection dialog.
```

Comment:
I think we should try to unify around the term Connection Dialog instead of Custom Dialog, since we now have V1 and V2, but this could be follow-up.

Reply:
added a tfs item to my queue to resolve.